components/Tabs/src/TabBarView/private/MDCTabBarViewItemView.m

```diff
@@ -132,4 +134,14 @@ - (CGSize)sizeThatFits:(CGSize)size {
   return CGSizeMake(width, height);
 }
 
+#pragma mark - Accessibility
+
+- (NSString *)accessibilityLabel {
+  return _accessibilityLabel ?: self.titleLabel.text;
+}
+
+- (void)setAccessibilityLabel:(NSString *)accessibilityLabel {
+  _accessibilityLabel = accessibilityLabel;
```

Comment:
I believe this setter is not necessary, since this property is settable on UIView by default.

Reply:
done